drivers/wifi/nrfwifi/src/fmac_main.c

```diff
@@ -252,8 +252,8 @@ static void nrf_wifi_process_rssi_from_rx(void *vif_ctx,
 
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
 
-	if (!rpu_ctx_zep) {
-		LOG_ERR("%s: rpu_ctx_zep is NULL", __func__);
+	if (!rpu_ctx_zep || !rpu_ctx_zep->rpu_ctx) {
```

Comment:
At least for me, I need to twist my brain when seeing multiple negations. We could write this as
`if (!(rpu_ctx_zep && rpu_ctx_zep->rpu_ctx)) {`
which is slightly more readable.